lsp-elm.el

```diff
@@ -0,0 +1,62 @@
+;;; lsp-elm.el --- Elm Client settings             -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019 Daniel V
+
+;; Author: Daniel V
+;; Keywords: elm lsp
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; lsp-elm client
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-elm nil
+  "LSP support for the Elm programming language, using the server from https://github.com/elm-tooling/elm-language-server"
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/elm-tooling/elm-language-server"))
+
+(defcustom lsp-elm-server-install-dir
+  (locate-user-emacs-file "elm-language-server/")
+  "Install directory for elm-language-server.  A slash is expected at the end.
+This directory should contain a clone of elm-tooling/elm-language-server from github."
+  :group 'lsp-elm
+  :risky t
+  :type 'directory)
+
+(defun lsp-elm--elm-command ()
+  "Generate LSP startup command for the Elm Language Server."
+  `("node" ,(concat (file-truename lsp-elm-server-install-dir)
```

Comment:
Use f-join and remove the comment about `A slash is ...`. f-join will handle that.

Reply:
Should be good now